### ISSUE TITLE
[Request] Re-instate on plugin hub for use during Leagues 5

### ISSUE THREAD

**Issue by Defiect** [geheur/trailblazer-attackspeed-fix#1]
This plugin was great last year, but the janky animations are back again in leagues 5. Can we get this plugin back on the plugin hub?

**Comment by geheur** [comment]
are you talking about mage animations? I haven't played leagues yet.

**Comment by Defiect** [comment]
> are you talking about mage animations? I haven't played leagues yet.

Yes, various animations that are likely included in your set from last time, e.g. casting ancients, shooting a crossbow

Similar to last league, these attacks speeds are seen going from 5t to 2t (in some situations, they are even seen at a 1t rate, such as when attacking echo sol totems)

Glanced at the plugin main, doesn't seem like any logic would need to be changed to support this league
